src/metax_api/services/file_service.py

```diff
@@ -246,7 +246,7 @@ def get_datasets_where_file_belongs_to(cls, file_identifiers):
         return Response(preferred_identifiers, status=status.HTTP_200_OK)
 
     @classmethod
-    def get_detailed_datasets_where_file_belongs_to(cls, file_identifiers):
+    def get_detailed_datasets_where_file_belongs_to(cls, file_identifiers, keysonly):
         """
         Find out which (non-deprecated) datasets a list of files belongs to, and return
```

Comment:
I don't know if this was once returning only non-deprecated datasets but at least I get deprecated datasets in return json